wordpress-mobile/gutenberg-mobile
# Fix merged of text blocks when a style is active.

Fixes #1873 

To test:
 - Start the demo app
 - Add a paragraph block
 - Write some text on it
 - Activate the Bold or Italic style
 - Write some more text
 - Move the cursor just after the style
 - Split the paragraph ( Press Enter)
 - Move the cursor to the middle of the word with style
 - Start to tap the delete key
 - Check the style characters are deleted and when getting to the start of the paragraph it's merged with the one above.

PR submission checklist:
